src/app/components/client/exposure_card/ExposureCard.stories.tsx

```diff
@@ -3,22 +3,67 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import type { Meta, StoryObj } from "@storybook/react";
-import { ExposureCard } from "./ExposureCard";
+import { Exposure, ExposureCard } from "./ExposureCard";
 import FamilyTreeImage from "../assets/familytree.png";
 import TwitterImage from "../assets/twitter-icon.png";
 import {
   createRandomBreach,
   createRandomScanResult,
 } from "../../../../apiMocks/mockData";
+import { StaticImageData } from "next/image";
+import { ReactNode } from "react";
+import { FeatureFlagName } from "../../../../db/tables/featureFlags";
+import {
+  defaultExperimentData,
+  ExperimentData,
+} from "../../../../telemetry/generated/nimbus/experiments";
+
+type ExposureCardWrapperProps = {
+  exposureData: Exposure;
+  exposureImg?: StaticImageData;
+  locale?: string;
+  isPremiumUser?: boolean;
+  isEligibleForPremium?: boolean;
+  resolutionCta?: ReactNode;
+  isExpanded?: boolean;
+  enabledFeatureFlags?: FeatureFlagName[];
+  experimentData?: ExperimentData;
+  removalTimeEstimate?: number;
+  onToggleExpanded?: () => void;
+};
+
+const ExposureCardWrapper = (props: ExposureCardWrapperProps) => {
+  return (
+    <ExposureCard
+      exposureImg={props.exposureImg}
+      exposureData={props.exposureData}
+      locale={props.locale ?? "en"}
+      isPremiumUser={props.isPremiumUser ?? false}
+      isEligibleForPremium={props.isEligibleForPremium ?? false}
+      resolutionCta={props.resolutionCta ?? null}
+      isExpanded={props.isExpanded ?? true}
+      enabledFeatureFlags={props.enabledFeatureFlags ?? []}
+      experimentData={
+        props.experimentData ?? {
+          ...defaultExperimentData,
+          "data-broker-removal-time-estimates": {
+            enabled: true,
+          },
+        }
+      }
+      removalTimeEstimate={props.removalTimeEstimate}
+      onToggleExpanded={props.onToggleExpanded ?? (() => {})}
+    />
+  );
+};
 
-// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction
-const meta: Meta<typeof ExposureCard> = {
+const meta: Meta<typeof ExposureCardWrapper> = {
   title: "ExposureCard",
-  component: ExposureCard,
+  component: ExposureCardWrapper,
   tags: ["autodocs"],
```

Comment:
It's unclear to me why this wrapper component would be needed? Is the goal just to pass in default properties? If so, I think you can just do that via the `args` property on `meta`?

```suggestion
  tags: ["autodocs"],
  args: {
    enabledFeatureFlags: [],
  },
```

Reply:
Well, it’s unclear to me as well — [41df155](https://github.com/mozilla/blurts-server/pull/5189/commits/41df1559d068cde9c9795e6e7546e15066871aa9). :D